### ISSUE TITLE
Draft a document on procedure to create Working Group/Task Force creation

### ISSUE THREAD

**Issue by shigeya** [bgin-global/gwg#2]

**Comment by tkns1192** [comment]
I will drop my random thoughts below...

1. Developments so far
As of now, all the working groups, study groups, and task-forces are created through the discussion during the block meetings. However, we should prepare additional paths to set up these groups in order to be agile enough to keep up with new developments in this ecosystem. 

2. Question regarding different types of groups
According to my understanding, there is no common understanding of the characteristics of different types of groups and their hierarchical relationships. Thus, I would like to propose to define the following tentative list of groups within BGIN;
1) Task Force: given mandates to develop specific outputs within a certain period of time and thereafter planned to be disbanded
2) Discussion Group: Intends no specific output but just gather to discuss the certain topic(s)
3) Study Group: Conducts research/study to decide whether BGIN should develop tangible outputs
4) Working Group: Intends to produce outputs related to certain themes and has no sunset plans

I have no good idea to define hierarchical relationships between them... Maybe, we should start from the discussion group and decide to upgrade it to a study group, some of which may become a working group? This procedure should be flexible (i.e. we can directly develop a working group depending on circumstances).  

3. Resource constraints
I also think that we should be mindful of resource constraints when developing new groups so that BGIN can develop tangible outputs within the intended time frame. Currently, we hear lots of proposals during the block meetings; however, there is not enough capacity to manage so many different groups. In the decision-making process to develop new groups, we certainly need to include criteria related to resource allocation.

**Comment by shigeya** [comment]
I think we can take hints from IETF-style WG creation steps.

Initially, people who have an interest get together to have unofficial BoF (called Bar BoF). Then, "official" BoF at IETF meetings multiple times. If there are interests, a WG charter is drafted and finally establishing WGs. Independently, people may create mailing lists to exchange thoughts.

I think SG may have similar functionality to the "official" BoF at IETF. We may need unofficial BoF, but at our block meetings, we have unconference sessions. Unfortunately, since we predetermine the topics of unconference sessions in the past meetings, we may need to adjust how we have BoF sessions at the block meetings.

On the constraint. If there is some leader/chair who has enough willingness to run a group, It's ok to create a group. 
Of course, we need to rely on the leader's capacity.

**Comment by tkns1192** [comment]
Yes. We can translate into;
Discussion group = Unofficial BoF
Study Group = Official BoF
I have no strong views on the name of the groups. 
Do you think BoF is a common word? I have never heard at least in the regulatory community.

**Comment by shigeya** [comment]
It's not common.

I've been believing it's CS community word (since I saw the word at a conference first, as I remember, at USENIX conferences and some private conferences in late 1980) but due to Wikipedia, it's IETF origin. That's different from my memory, but maybe so.

https://en.wikipedia.org/wiki/Internet_Engineering_Task_Force#Organization

**Comment by tkns1192** [comment]
Thank you for the explanation. Then, my preference is to use plain words to let everyone easily understand the meaning.

**Comment by shigeya** [comment]
Agreed. 

btw, I feel "discussion group" more official than "study group" (my feelings might be wrong. Not sure)
I think "Study Group" is good, but need a different word for "Discussion group"

Any alternative words? I also think about alternatives.

**Comment by tkns1192** [comment]
How about the following?
a) Study Group
b) Work Stream
c) Working Group

**Comment by shigeya** [comment]
I believe we already have multiple workstreams inside a working group. Is the above structure fit with the current structure?

**Comment by tkns1192** [comment]
Good point... IMHO, sub-groups within working groups could be named differently. Maybe just sub-group on XXX?

**Comment by shigeya** [comment]
Maybe, but I feel "workstream" is a natural name for a subgroup of any working group.

I'm not fun of keeping this "name game" going :<   Let us find some solution quickly.
My proposal at this moment is - forget about "official" pre-WG groups.

- "Study Group" for unofficial study group which tries to launch as a working group. Once the group established enough interest, the group will prepare for a Working Group Charter. 
- "Working Grop" launch after approval (from community OR steering committee? I'm not sure) of the charter
- WG members can establish "Work Streams" as necessary

**Comment by tkns1192** [comment]
OK

**Comment by tkns1192** [comment]
A very high-level idea for the process would be;
1) Members can propose establishing a study group at any point through mailing lists, Zulip chat, and block meetings. If two or more people show interest, they can start a study group. They need to assign at least one chair. When they want to have someone as a secretariat, they need to consult with Steering Committee or ask during a block meeting for resource constraints reasons. Terms of reference/charter are optional.
2) Study group chair(s) can propose to upgrade the study group to a working group during block meetings. In that case, they need to propose terms of reference/charter of the group for approval. (Only a block meeting resolution can approve the establishment of a working group)
3) Members can propose to establish a working group at a block meeting directly without through a study group period. In that case, they need to propose terms of reference/charter for approval.

**Comment by shigeya** [comment]
It looks very nice. Thank you.

One of the questions I have is whether we have to wait for block meetings to establish working groups.  If the proposal is openly available, gives a good amount of attention to the community, and also with enough amount of time for think (two to three weeks?), we don't need to rely on block meetings. 

We may able to use block meetings to *shorten* the process, but I think waiting for several months is not necessary.

**Comment by tkns1192** [comment]
Since the working group does not need to have a sunset plan, it will consume resources for an unspecified period of time. This means that we need to set a relatively high bar to establish one. Thus, my preference is to rely on block meetings for approval. However, depending on the interval between block meetings and a broader discussion on the decision-making process of BGIN, I can go along with the different processes.

**Comment by shigeya** [comment]
I understand your concern. We need to make the bar higher enough.

By the way, all of the WG at IETF and W3C have a deadline. For example, [DID WG Charter at W3C](https://www.w3.org/2020/12/did-wg-charter.html)

Note, there is an `FTE` entry in the charter -- The amount of full-time equivalent HR required for supporting the working group. `Team` is a W3C term for W3C staff.

**Comment by tkns1192** [comment]
I can understand the idea to include the sunset clause in the charter of all the groups within BGIN although I also concern that it may not be easy to decide when to sunset some of the working groups such as IGWG. Regardless of the sunset clause, I believe we should assign enough resources including secretariats to the working groups and thus need to set a high bar in order not to dissipate the focus. 

All in all, for the draft bylaws, my preference is not to require the sunset clause for all the groups and to allow only block meetings to approve the establishment of working groups.  Even study groups can develop documents and other output, which would allow us to be agile enough to respond to the abrupt events.

**Comment by shigeya** [comment]
Ok. Sounds good.

For abrupt events, such as what we discussed in the multistakeholder governance study paper, we may need some mechanism, but for any documents, what you discussed seems enough.

**Comment by shigeya** [comment]
Tasks from IGWG call on April 27th
- [x] setup a call after May 10th.
- [ ] include experience from the bylaws task force
- [ ] experimenting by creating a `treasury`  group

**Comment by shigeya** [comment]
Document template is here: https://github.com/bgin-global/igwg/blob/master/Guidance/Drafts/Tentative%20guidance%20on%20the%20creation%20of%20the%20BGIN%20document.docx

**Comment by shigeya** [comment]
As a side note, IETF clarified the support level of side meetings. Maybe, we need this sort of clarification in the future.

https://www.ietf.org/blog/public-side-meetings/
